backstop/scenarios/select.js

```diff
@@ -4,25 +4,30 @@ module.exports = [
   // Select types
   scenario('Type Default', {
     selectors: ['#demo-default'],
+    viewports: [{ width: 800, height: 600 }],
```

Comment:
So you have changed viewport and as a result there are new images, which now is hard to compare with previous.
But also previous one should be removed then, no?

Reply:
We have script to remove old images, might help..
`yarn clean:images`

Reply:
Removed old test images.  I didn't think that would happen since I didn't change the name of the scenario.

Reply:
Now I see.. It got appended `desktop` at the end previously in images names, now it was removed after you changed viewport...
Silly situation, ideally would be good to keep same images name so it could be easier to compare the visuals :/

Also, tests are failing, cause for some reason you pushed new user icon images.

Reply:
I see we have this default viewport change in multiple components already.. and we are not using any labels like the default one:
```js
{ label: 'desktop',  width: 1920, height: 1080 },

If we ever gonna start testing responsiveness, this might be important and we would need to refactor all of this. Set a list of accepted viewports and label them, for example.
But I guess it is not this PRs scope, sometime in the future...